ThePensionsRegulator.Frontend/TagHelpers/TprSearchTagHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System.Threading.Tasks;
+using ThePensionsRegulator.Frontend.HtmlGeneration;
+
+namespace ThePensionsRegulator.Frontend.TagHelpers
+{
+    [HtmlTargetElement(TagName)]
+    public class TprSearchTagHelper : TagHelper
+    {
+        internal const string TagName = "tpr-search";
+
+        private readonly ITprHtmlGenerator _htmlGenerator;
+
+        [HtmlAttributeName("faq-api-url")]
+        public string FaqApiUrl { get; set; } = string.Empty;
```

Comment:
`FaqApiUrl` is different to the page title `Q&As search results`. It also locks this component into a FAQ-like content, when all your other naming in code suggests it's more generic than that.

Reply:
I have added and renamed attributes to make them more generic for a questions and answers component